default.lcf

```diff
@@ -107,7 +107,8 @@ log4j.category.javax.jmdns=OFF
 # log4j.category.jmri.jmrit.symbolicprog.tabbedframe=DEBUG
 # log4j.category.jmri.jmrit.symbolicprog.DecVariableValue=DEBUG
 # log4j.category.jmri.jmrix=DEBUG
-# log4j.category.jmri.jmrix.can=DEBUG
+log4j.category.jmri.managers.DefaultShutDownManager=DEBUG
```

Comment:
Could you remove these two DEBUG lines, please?

Reply:
oops sorry